### DISCUSSION TITLE
add existing items to collection

### DISCUSSION THREAD

**Discussion by m00nlygreat** [anyproto/contributors#152] [Q&A] [answered]
I love the concept of Anytype. Thanks for great app

Regarding Collections, while the documentation describes them as folder-like structures, do I need to create all items within Collections? If they function as folders, can I simply create symbolic links within them instead?

**Comment by fuksman** [comment, marked as answer]
Hey. Yes, it's possible. Click the three-dots icon on the top right, choose the "Add link to object" option, and then select the collection you would like to use.

**Reply by m00nlygreat** [reply]
That's great! Thanks for the reply
